docs/extensions/civix.md

```diff
@@ -8,7 +8,7 @@ community-endorsed method for building your CiviCRM extensions.
 Follow the installation instructions in the
 [GitHub repository](https://github.com/totten/civix/).
 
-In order to verify that all your configuration is correct ping CiviCRM from
+After fulfilling the [Pre-Requisites](/extensions/basics/#pre-requisites), you can verify that all your configuration is correct using ping CiviCRM from
```

Comment:
I like this change conceptually, but feel the grammar could use some improvement. I suggest replacing "using ping CiviCRM" with "by running the following command". Can you make this adjustment?